main.js

```diff
@@ -160,6 +160,9 @@ bot.addListener("message", function(from, to, text, message) {
 		}
 		else bot.action(config.channels[0], "dabs on "+from +" "+randomFromArray(dabEmojis));
 	}
+	if((message.args[1].toLowerCase().indexOf("succ") > -1){
+		bot.say(config.channels[0], "S U C C");
```

Comment:
are you making a command, or a response to a trigger word?

Reply:
response to a trigger word

Reply:
@JordanReimers pls

Reply:
approve me you coward